integration-tests/src/test/resources/indexer/wikipedia_cloud_simple_index_task.json

```diff
@@ -0,0 +1,81 @@
+{
```

Comment:
Hadoop InputSource has a bug with batch parallel so using this non parallel for now.